deployment-bundle-steps.md

```diff
@@ -0,0 +1,135 @@
+# How to setup a project with deployment to S3
+
+1. Create S3 bucket for production / staging (your-bucket-name-production & your-bucket-name-staging). (If using cloudfront, see [optional settings](#optional-settings))
```

Comment:
no I think Infrastructure have a rule about this so check with @matt-smith for example and then you can tighten this up, probably automate it.